recipe/conda_build_config.yaml

```diff
@@ -289,6 +289,8 @@ pin_run_as_build:
     max_pin: x.x.x
   sqlite:
     max_pin: x
+  tinyxml2:
+    max_pin: x.x
```

Comment:
`tinyxml2` has a `run_exports` https://github.com/conda-forge/tinyxml2-feedstock/blob/8895462aa37083e7c32d676252887b7b7baba1f0/recipe/meta.yaml#L17 so this is not needed.
```suggestion
```